docs/snippets/common/foo-bar-baz-story.js.mdx

```diff
@@ -1,9 +1,12 @@
 ```js
 // FooBar.stories.js | FooBar.stories.jsx | FooBar.stories.ts | FooBar.stories.tsx
 
+import { Foo } from './Foo';
+
 export default {
+  component:Foo,
```

Comment:
```suggestion
  component: Foo,
```

Reply:
Updated with the latest commit. Marking it as solved. Thanks for catching this